packages/core/src/components/pane/pane.tsx

```diff
@@ -0,0 +1,620 @@
+      }
+    }
+  }
+
+  @Watch('size')
+  onSizeChange() {
+    if (this.expanded) {
+      if (this.isMobile) {
+        this.animateHorizontalFadeIn('100%');
+      } else {
+        const expandPaneSize = this.getExpandPaneSize();
+
+        if (this.isBottomTopPane) {
+          this.animateHorizontalFadeIn(expandPaneSize);
+        } else {
+          this.animateVerticalFadeIn(expandPaneSize);
+        }
+      }
+    } else {
+      this.showContent = false;
+
+      if (this.isMobile) {
+        this.resetMobileExpanded();
+      } else {
+        if (this.isBottomTopPane) {
+          this.animateHorizontalFadeIn(this.collapsedPane);
+        } else {
+          this.animateVerticalFadeIn(this.collapsedPane);
+        }
+      }
+    }
+  }
+
+  render() {
+    return (
+      <Host
+        class={{
+          'inline-color': !this.floating,
+          'mobile-overlay': this.expanded && this.isMobile,
+          'top-expanded': this.expanded && this.isMobileTop && this.isMobile,
+          'bottom-expanded':
+            this.expanded && !this.isMobileTop && this.isMobile,
+          'top-bottom-pane': this.isBottomTopPane && !this.isMobile,
+          'left-right-pane': this.isLeftRightPane && !this.isMobile,
+          [`${this.composition}-pane-border`]:
+            !this.borderless && !this.isMobile && !this.floating,
+          'nav-left-border':
+            !this.borderless &&
+            !this.isMobile &&
+            this.composition !== 'right' &&
+            this.floating,
+          'mobile-border-top':
+            !this.borderless &&
+            this.isMobileTop &&
+            this.isMobile &&
+            !this.expanded &&
+            !this.floating,
+          'mobile-border-bottom':
+            !this.borderless &&
+            !this.isMobileTop &&
+            this.isMobile &&
+            !this.expanded &&
+            !this.floating,
+          'box-shadow': this.floating,
+          'aria-expanded': this.expanded,
+          'not-visible': this.hideOnCollapse && !this.expanded,
+        }}
+      >
+        <aside
+          class={{
+            'top-bottom-pane': this.isBottomTopPane && !this.isMobile,
+            'left-right-pane': this.isLeftRightPane && !this.isMobile,
+            'mobile-pane': this.isMobile,
+            expanded: this.expanded,
+          }}
+        >
+          <div
+            id="title-div"
+            class={{
+              title:
+                !this.isMobile && !this.hideOnCollapse && !this.showContent,
+              'title-finished':
+                !this.isMobile && !this.hideOnCollapse && this.showContent,
+              'title-expanded':
+                !this.isMobile && !this.hideOnCollapse && this.expanded,
+              'title-hide-on-collapse': !this.isMobile && this.hideOnCollapse,
+              'title-mobile': this.isMobile,
+              'header-gap': !this.isMobile && !this.hideOnCollapse,
+            }}
+          >
+            <ix-icon-button
```

Comment:
This button can be focused even if it lies behind another pane and is therefore not visible on screen.